ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/services/ServerInternalKsqlClient.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.server.services;
+
+import static java.util.Objects.requireNonNull;
+
+import io.confluent.ksql.rest.client.KsqlClientUtil;
+import io.confluent.ksql.rest.client.RestResponse;
+import io.confluent.ksql.rest.entity.KsqlEntityList;
+import io.confluent.ksql.rest.entity.KsqlRequest;
+import io.confluent.ksql.rest.server.resources.KsqlResource;
+import io.confluent.ksql.services.ServiceContext;
+import io.confluent.ksql.services.SimpleKsqlClient;
+import java.net.URI;
+import java.util.Collections;
+import javax.ws.rs.core.Response;
+
+public class ServerInternalKsqlClient implements SimpleKsqlClient {
```

Comment:
nit: I think it'd be good to have a brief javadoc explaining why we need an internal client and what it's used for (using it in the heath check resource to hit the KsqlResource).